bin/melon.js

```diff
@@ -30,89 +30,11 @@ program
   });
 
 program
-  .command('deploy')
+  .command('deploy <endpoint>')
   .description('Deploy the Melon smart contracts')
-  .action(async (dir, cmd) => {
-    const {
-      initTestEnvironment,
-    } = require('../lib/utils/environment/initTestEnvironment');
-    const { deploySystem } = require('../lib/utils/deploySystem');
-    const environment = await initTestEnvironment();
-    const thisDeployment = await deploySystem();
-    const deploymentsPath = path.join(
-      __dirname,
-      '..',
-      'out',
-      'deployments.json',
-    );
+  .action(async (endpoint) => {
+    console.error('Deployment is currently not implemented. Tests now run on the in-memory devchain.');
 
-    let otherDeployments = {};
-
-    fs.access(deploymentsPath, fs.constants.F_OK | fs.constants.W_OK, err => {
-      if (err) {
-        console.error(
-          `${deploymentsPath} ${
-            err.code === 'ENOENT' ? 'does not exist' : 'is read-only'
-          }`,
-        );
-      } else {
-        const raw = fs.readFileSync(deploymentsPath, { encoding: 'utf8' });
-        otherDeployments = JSON.parse(raw);
-      }
-    });
-
-    const deploymentId = `${await environment.eth.net.getId()}:${
-      environment.track
-    }`;
-
-    otherDeployments[deploymentId] = thisDeployment;
-
-    fs.writeFileSync(
-      deploymentsPath,
-      JSON.stringify(otherDeployments, null, 2),
-    );
-    console.log(
-      'Wrote deployed addresses as',
-      deploymentId,
-      'to',
-      deploymentsPath,
-    );
-    console.log(
-      "You can use it with: `import protocol from '@melonproject/protocol';",
-    );
-    console.log('// and then ...;');
-    console.log(
-      'const deployment = protocol.utils.solidity.getDeployment(environment);',
-    );
-    process.exit();
-  });
-
-program
-  .command('set-price <symbol> <value>')
```

Comment:
Because with the in-memory chain these comamnds should be built in the frontend, not in the protocol. Because the in-memory chain cannot be affected by commands from the outside (because it is not exposed on the outside).

Reply:
Commands like this will also make sense for the deployed versions on kovan/mainnet, not only in-memory.